app/helpers/report_helper.rb

```diff
@@ -146,4 +146,129 @@ def reports_group_title
          :group_description => User.current_user.current_group.description}
     end
   end
+
+  def dashboard_widget_summary(widget, sb_data)
+    schedule = widget.miq_schedule
+    timezone = schedule&.run_at ? schedule.run_at[:tz] : session[:user_tz]
+    safe_join([
+                widget_basic_information(widget),
+                widget_status(widget, sb_data, timezone),
+                widget_sb_type_report(widget, sb_data),
+                widget_timer(schedule, timezone, sb_data),
+                widget_visibility(widget),
+              ])
+  end
+
+  private
+
+  def summary_data(title)
+    {:title => title, :mode => "miq_widget_#{title.downcase.tr(' ', '_')}"}
+  end
+
+  def row_data(label, value, style = "")
+    data = {:cells => {:label => label, :value => value}}
+    data[:style] = style if style.present?
+    data
+  end
+
+  def widget_basic_information(widget)
+    read_only = widget.read_only ? '* ' : ''
+    data = summary_data(_('Basic Information'))
+    data[:rows] = [
+      row_data(_('Title'), "#{read_only}#{widget.title}"),
+      row_data(_('Descrption'), "#{read_only}#{widget.description}"),
+      row_data(_('Active'), widget.enabled),
+      row_data(_('Default'), widget.read_only)
+    ]
+    miq_structured_list(data)
+  end
+
+  def widget_status(widget, sb_data, timezone)
+    return if sb_data[:wtype] == "m"
+
+    data = summary_data(_('Status'))
+    glyphicon = widget.decorate.try(:secondary_icon)
+    current_status = glyphicon ? [{:icon => glyphicon, :value => widget.status}] : widget.status
```

Comment:
`widget.status ` i feel can also be / should be translated, since its results will be things like `completed` which i'm fairly certain already exist in our translation file.